go/tasks/pluginmachinery/flytek8s/copilot.go

```diff
@@ -63,23 +63,24 @@ func CopilotCommandArgs(storageConfig *storage.Config) []string {
 		"/bin/flyte-copilot",
 		"--storage.limits.maxDownloadMBs=0",
 	}
-	if len(storageConfig.Stow.Config) > 0 && len(storageConfig.Stow.Kind) > 0 {
-		var cfg string
-		for key, val := range storageConfig.Stow.Config {
-			cfg += fmt.Sprintf("%s=%s,", key, val)
-		}
-		commands = append(commands, []string{
-			fmt.Sprintf("--storage.stow.config %s", cfg),
-			fmt.Sprintf("--storage.stow.kind=%s", storageConfig.Stow.Kind),
-		}...)
-	}
 	if storageConfig.MultiContainerEnabled {
 		commands = append(commands, "--storage.enable-multicontainer")
 	}
+	if len(storageConfig.InitContainer) > 0 {
+		commands = append(commands, fmt.Sprintf("--storage.container=%s", storageConfig.InitContainer))
+
+	}
+	commands = append(commands, fmt.Sprintf("--storage.type=%s", storageConfig.Type))
+
+	if storageConfig.Type == storage.TypeStow {
+		var cmd = ""
```

Comment:
You should keep the check for `if len(storageConfig.Stow.Config) > 0 && len(storageConfig.Stow.Kind) > 0 {` here as well because there is legacy way and new way of configuring stow... and having a Stow.Config is the "new" way...